Pokemon Tabletop Adventure/Data Models/WBPokémon.swift

```diff
@@ -18,7 +18,7 @@ struct Pokémon: Codable, Equatable {
     let stats: StatArray
     let type: [PokeType]
     let abilities: PokeAbilities
-    let evolution: [String: Int]
+    let evolution: [String: String]
```

Comment:
This is gonna make some things more of a pain in the arse, but hopefully it'll actually not - I'm expecting we can just display the evolution requirements as a plain label.